src/proteus/dswx_hls.py

```diff
@@ -3189,10 +3268,15 @@ def generate_dswx_layers(input_list,
     logger.info(f'    product version: {product_version}')
     logger.info(f'    software version: {SOFTWARE_VERSION}')
     logger.info(f'processing parameters:')
-    logger.info(f'    flag_use_otsu_terrain_masking: {flag_use_otsu_terrain_masking}')
-    logger.info(f'    min_slope_angle: {min_slope_angle}')
-    logger.info(f'    max_sun_local_inc_angle: {max_sun_local_inc_angle}')
-    logger.info(f'    mask_adjacent_to_cloud_mode: {mask_adjacent_to_cloud_mode}')
+    if flag_use_otsu_terrain_masking:
```

Comment:
Uhm, I don't think this is left over from the cast shadow masking. We added this flag in the runconfig in the cal/val or cal/val point release. We can change it to enum in the future, but that's not the goal here.

Reply:
Oh, taking a step back, I think I see now what the goal of this change is: the desire to be more descriptive for the user in the log message.

I agree that this is a good goal. However, this section seems to be meant to describe the processing parameters; during development, I genuinely appreciated that the exact parameter names and values were reported. By adding in additional logic here, it obscures those actual input parameters. For example, it displays a computed `terrain_masking_algorighm` instead of the actual parameter, which is `flag_use_otsu_terrain_masking`.

Could you please revert this to how it was before, where all parameters are logged? If more clarity is needed to be logged for the user, then perhaps log the computed name of the algorithm at a later point?

![Screen Shot 2022-12-22 at 2 00 02 PM](https://user-images.githubusercontent.com/11642807/209233325-8ecdabf5-8479-4342-98f7-7bf4ec784ed2.png)

Reply:
Hi @nemo794 , the idea here is that `min_slope_angle` and `max_sun_local_inc_angle` are only used by shadow masking without Otsu.

Reply:
I understand, but they are read in as parameters regardless, and should be logged accordingly.

Reply:
Uhm why? To be safe? I'd prefer not to overload the user with unnecessary information.

Reply:
The user will already have filled out the runconfig and have considered that information, so it is not extra burden for them.

Reporting it here is safe; plus, this section has personally helped me with debugging already. It's very useful to get an exact readout of all parameters that the function is using.

I do like the idea of logging the actual name of which method will be used because that is opaque in the code, but that can be done afterwards. If it's done afterwards, it'd be great to also announce if OTSU was chosen (currently, this code skips that case).

Reply:
The users don't know the details of the shadow masking algorithms, and they may get confused about why we are using some parameters when they are not even used. I updated the messages to show the parameters regardless if they are used or not, and I'm making it clear when the parameters are not used:
```
    terrain masking algorithm: Otsu
        min. slope angle: -5  (unused)
        max. sun local inc. angle: 40 (unused)
```
or
```
    terrain masking algorithm: Sun local incidence angle
        min. slope angle: -5
        max. sun local inc. angle: 40
```

Reply:
This looks clearer - thank you! A minor follow-up suggestion below: [link](https://github.com/nasa/PROTEUS/pull/32/files#r1056637957).